File: 260127/정상적인 구구단/normal-multiple.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);
        int N=sc.nextInt();

        for(int i=1; i<=N; i++){
            for(int j=1; j<N+1; j++){
                if(j==N){
                    System.out.printf("%d * %d = %d\n",i,j,i*j);
                    continue;
                }
                System.out.printf("%d * %d = %d, ",i,j,i*j);
            }
        }
    }
}

File: 260303/정상적인 구구단/normal-multiple.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int n;
    cin>>n;

    for(int i=1; i<n+1; i++){
        for(int j=1; j<n+1; j++){
            printf("%d * %d = %d",i,j,i*j);
            j==n?cout<<endl:cout<<", ";
        }
    }
    return 0;
}